firmware/application/ui_navigation.cpp

```diff
@@ -156,6 +160,9 @@ SystemStatusView::SystemStatusView(
     refresh();
 
     button_back.on_select = [this](ImageButton&) {
+        if (portapack::persistent_memory::save_load_pmem_from_sdcard_flag()) {
+            portapack::persistent_memory::save_persistent_settings_to_file("SETTINGS/pmem_settings");
```

Comment:
nit: consider a constant for that string. It's used in quite a few places.

Reply:
Did both use a define for those files and removed the save and load parameter